### PR TITLE
Add permitted actions support for parentship functionality

### PR DESCRIPTION
**Pull request by patari**
#### Summary

Fixes create/update/delete button visibility.

### CODE REVIEW

File: frontend/src/employee-frontend/components/person-profile/PersonFridgeChild.tsx [hunk 1]
```diff
@@ -109,60 +111,70 @@ export default React.memo(function PersonFridgeChild({ id, open }: Props) {
...
+                          setSelectedParentshipId(fridgeChild.id)
+                          toggleUiMode(`edit-fridge-child-${fridgeChild.id}`)
+                        }}
+                        deletable={permittedActions.includes('DELETE')}
+                        onDelete={() => {
+                          setSelectedParentshipId(fridgeChild.id)
+                          toggleUiMode(`remove-fridge-child-${fridgeChild.id}`)
+                        }}
+                        conflict={fridgeChild.conflict}
+                        deletableFor={
```
Comment (Gekkio) [inline frontend/src/employee-frontend/components/person-profile/PersonFridgeChild.tsx:168]: Is this still needed? What happens when both `deletable` and `deletableFor` are set?

Comment (patari) [reply frontend/src/employee-frontend/components/person-profile/PersonFridgeChild.tsx]: Removed this. Also fixed bug in `deletable` based on what's actually happening in `ParentshipController`.

**Review by Gekkio — APPROVED** [review]

**Comment by Gekkio** [comment]
Needs rebase :scream:

**Comment by patari** [comment]
> Needs rebase scream

Done!

**Pull request merged**